permissions/plugin-matlab.yml

```diff
@@ -7,3 +7,6 @@ paths:
   - "org/jenkins-ci/plugins/matlab"
 developers:
   - "mathworks_ci_team"
+security:
+  contacts:
+    jira: "mathworks_ci_team"
```

Comment:
this isn't needed if you only have one user `mathworks_ci_team` will already be the contact

Reply:
Ok thanks @timja shall i close this PR then ?

Reply:
yes there's no need for it